lib/esm/ModuleChunkFormatPlugin.js

```diff
@@ -113,7 +113,7 @@ class ModuleChunkFormatPlugin {
 									// create final path
 									return (
 										(baseOutputName.length > 0
-											? "../".repeat(baseOutputName.length)
+											? "../".repeat(baseOutputName.length - 1)
```

Comment:
length needs to be offset since repeat(0) would still place one `../`, when using length you are always getting an extra `../`